spec/compiler/crystal/tools/format.cr

```diff
@@ -0,0 +1,291 @@
+require "spec"
```

Comment:
Should this file be named with `_spec.cr` at the end, otherwise it won't run the specs?

Reply:
This spec is ran. (See spec/compilr_spec.cr.) But this file name should be `format_spec.cr` conventionally. I'll fix it.